hangman/lib/hangman/client.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Hangman.Client do
+  
+  def new_game() do
```

Comment:
# -3

This is going to sound picky, but it isn't—the server should not have a `new_game` handler. Here's why.

A server has just one job:™ it needs to maintain state. In OO, it's called the class invariant. Whenever that state leaks out, or is allowed to become invalid, you have a source of problems.

In this case, the server's state is the game returned by Game.new_game. But between the time the server is started and the time you call Client.new_game(), the server's state is `nil`. The server relies on the outside world calling things in the correct order.

Now instead define `Server.init/1`. It sets the state to a Game.new_game. This means that the server will always have valid state, and the `make_move` and `tally` calls will always work.